*** /usr/lib/python3.7/site-packages/matplotlib/__init__.py	2019-11-22 12:23:04.000000000 +0100
--- /usr/lib/python3.7/site-packages/matplotlib/__init__.py	2019-11-22 12:23:33.000000000 +0100
***************
*** 619,626 ****
          if not os.path.isdir(path):
              raise RuntimeError('Path in environment MATPLOTLIBDATA not a '
                                 'directory')
!         cbook.warn_deprecated(
!             "3.1", name="MATPLOTLIBDATA", obj_type="environment variable")
          return path
  
      def get_candidate_paths():
--- 619,626 ----
          if not os.path.isdir(path):
              raise RuntimeError('Path in environment MATPLOTLIBDATA not a '
                                 'directory')
! #        cbook.warn_deprecated(
! #            "3.1", name="MATPLOTLIBDATA", obj_type="environment variable")
          return path
  
      def get_candidate_paths():
